models/LogLevel.js

```diff
@@ -1,11 +1,10 @@
 module.exports = require('./Enum').create({
   values: [
-    'FATAL',
-    'ERROR',
-    'WARN',
-    'INFO',
-    'DEBUG',
-    'TRACE'
-  ],
-  autoLowerCase: true
```

Comment:
This should have been `autoUpperCase: true`, but it's used when a model is being coerced that has an Enum as a property.

**LogLevel.js**
```js
module.exports = require('./Enum').create({
  values: ['FATAL', ...] // All of these are uppercase
})
```

**Config.js**
```js
module.exports = require('./Model').extend({
  properties: {
    logLevel: LogLevel,
    ...
  }
})
```

**Creating the config**
```js
const config = new Config({
  logLevel: 'fatal' // Automatically gets converted to uppercase and works
})
```

Additionally, I would prefer that all Enum values be converted back to uppercase as that is the way they should be referenced. I do not want people referencing the values as: `LogLevel.fatal`. I would prefer: `LogLevel.FATAL` as it was before.